### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kjmrknsn**

### CODE REVIEW

**Comment by ajbozarth** [comment]
Nice catch, if you don't mind could you check for any other typos and fix them here as well? If you can't find anything in a day or two I'll merge this.

**Comment by kjmrknsn** [comment]
@ajbozarth Thanks for your comment and suggestion. I checked the whole website briefly and fixed other typos which I found.

File: site/docs/0.4.0-incubating/rest-api.html [hunk 1]
```diff
@@ -551,12 +551,12 @@ <h4 id="request-parameters-3">Request Parameters</h4>
   <tr>
     <td>from</td>
     <td>Offset</td>
-    <td>Offset</td>
+    <td>int</td>
```
Comment (ajbozarth) [inline site/docs/0.4.0-incubating/rest-api.html:554]: We fixed these in the Livy Docs (what this is built from), but it won't propagate until 0.4.1 so I'm ok updating this manually here for now, @jerryshao opinion?

**Review by ajbozarth — APPROVED** [review]
Thanks! LGTM, I'll give time for @jerryshao to respond to my comment before merging though.

File: site/docs/0.4.0-incubating/rest-api.html (hunk 1 above)
Comment (jerryshao) [reply site/docs/0.4.0-incubating/rest-api.html:554]: LGTM.

**Pull request closed without merging**